src/webapp/js/actions/index.js

```diff
@@ -380,3 +382,26 @@ export function fetchReports() {
     payload: request,
   };
 }
+
+export function fetchAutomatedReports() {
+  const url = '/api/job';
+  const request = apiClient.get(url);
+
+  return {
+    type: FETCH_AUTOMATED_REPORTS,
+    payload: request,
+  };
+}
+
+export function updateAutomatedReports(reportSettings, callback) {
+  return async (dispatch) => {
+    const url = 'api/job/update';
+    const result = await dispatch({
+      type: UPDATE_AUTOMATED_REPORTS,
+      payload: apiClient.put(url, reportSettings),
+    });
+    callback();
```

Comment:
in mots in all other places callback is called like this:
`const request = apiClient.put(${BASE_URL}/district/${values.id}, values);`
`  request.then(() => callback());`
`  return {`
`    type: SAVE_DISTRICT,`
`    payload: request,`
`  };`
so I think you should change it